docs/content/getting-started/hello-dagster.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Hello Dagster | Dagster
+description: Run dagster for the first time
+---
+
+# `hello dagster`
+
+This guide helps you build a simple data pipeline in dagster that downloads the top 10 HackerNews stories.
+
+1. Start by creating a file called `hello-dagster.py` and copying the following:
+
+    ```python file=/getting-started/hello-dagster/hello-dagster.py
+    # copy to a file called hello-dagster.py
+    from dagster import repository, asset, MetadataValue
+    import requests
+    import pandas as pd
+
+    # dagster helps you think about datasets, models, 
+    # and other objects that you want to exist as assets
+    # instead of worrying about tasks
+
+    @asset
+    def hackernews_topstory_ids():
+        """
+        Get up to 500 top stories from the HackerNews topstories endpoint.
+        API Docs: https://github.com/HackerNews/API#new-top-and-best-stories
+        """
+        newstories_url = "https://hacker-news.firebaseio.com/v0/topstories.json"
```

Comment:
Should `newstories_url` be `topstories_url`? Or inlined with the function call? And `top_500_newstories` be `top_500_stories`? I found myself trying to figure out what a "newstory" was.